awx/ui/src/screens/Organization/shared/OrganizationForm.js

```diff
@@ -72,7 +72,7 @@ function OrganizationFormFields({
           tooltip={t`The maximum number of hosts allowed to be managed by this organization.
             Value defaults to 0 which means no limit. Refer to the Ansible
             documentation for more details.`}
-          validate={minMaxValue(0, Number.MAX_SAFE_INTEGER)}
+          validate={minMaxValue(0,2147483647)}
```

Comment:
can you explain where this number comes from?

Reply:
nvm I see this is just max 32 bit integer

Reply:
Hello @fosterseth , Thanks for reviewing the code. I have removed the debug level as suggested and Kept the original license check in the code to ensure compliance.